testkit/src/main/scala/akka/kafka/testkit/internal/KafkaTestKit.scala

```diff
@@ -5,11 +5,12 @@
 
 package akka.kafka.testkit.internal
 
-import java.util
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicInteger
 import java.util.{Arrays, Properties}
 
+import scala.collection.JavaConverters._
```

Comment:
I have to re-learn as well, this should be `scala.jdk.CollectionConverters._` now.